src/VisualStudio/Core/Test/ClassView/SyncClassViewTests.vb

```diff
@@ -0,0 +1,661 @@
+        <Document>
+            Namespace N
+                Class C
+                    Sub $$M()
+                    End Sub
+                End Class
+            End Namespace
+        </Document>
+    </Project>
+</Workspace>
+
+            Await TestAsync(workspace,
+                 canonicalNodes:={
+                    Package("VBTestAssembly"),
+                    [Namespace]("N"),
+                    [Class]("C"),
+                    Member("M()")
+                 },
+                 presentationNodes:={
+                    Package("VBTestAssembly"),
+                    [Namespace]("N"),
+                    [Class]("C"),
+                    Member("M()")
+                 })
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.VsNavInfo)>
+        Public Async Function VisualBasic_TestMethod4() As Task
+            Dim workspace =
+<Workspace>
+    <Project Language="Visual Basic" CommonReferences="true" AssemblyName="VBTestAssembly">
+        <Document>
+            Namespace N
+                Class C
+                    $$Sub M()
+                    End Sub
+                End Class
+            End Namespace
+        </Document>
+    </Project>
+</Workspace>
+
+            Await TestAsync(workspace,
+                 canonicalNodes:={
+                    Package("VBTestAssembly"),
+                    [Namespace]("N"),
+                    [Class]("C"),
+                    Member("M()")
+                 },
+                 presentationNodes:={
+                    Package("VBTestAssembly"),
+                    [Namespace]("N"),
+                    [Class]("C"),
+                    Member("M()")
+                 })
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.VsNavInfo)>
+        Public Async Function VisualBasic_TestMethod5() As Task
+            Dim workspace =
+<Workspace>
+    <Project Language="Visual Basic" CommonReferences="true" AssemblyName="VBTestAssembly">
+        <Document>
+            Namespace N
+                Class C
+            $$        Sub M()
+                    End Sub
+                End Class
+            End Namespace
+        </Document>
+    </Project>
+</Workspace>
+
+            Await TestAsync(workspace,
+                 canonicalNodes:={
+                    Package("VBTestAssembly"),
+                    [Namespace]("N"),
+                    [Class]("C"),
+                    Member("M()")
+                 },
+                 presentationNodes:={
+                    Package("VBTestAssembly"),
+                    [Namespace]("N"),
+                    [Class]("C"),
+                    Member("M()")
+                 })
+        End Function
+
+#End Region
+
+        Private Async Function TestAsync(
+            workspaceDefinition As XElement,
+            Optional canonicalNodes As NodeVerifier() = Nothing,
+            Optional presentationNodes As NodeVerifier() = Nothing
```

Comment:
You're right. Checking the differences between canonical nodes and presentation nodes is really the job of the IVsNavInfo unit tests. I'll just check presentation for these tests.